### ISSUE TITLE
CSS “aspect-ratio: 1” is being incorrectly marked as an error

### ISSUE THREAD

**Issue by simevidas** [validator/validator#1406]
URL being validated or code to reproduce error: https://webplatform.news

```
    Error: CSS: aspect-ratio: 1 is not a aspect-ratio value.

    At line 106, column 17

     aspect-ratio: 1;↩}↩↩/* MAIN *
```

<img width="565" alt="Screenshot 2022-08-05 at 08 41 41" src="https://user-images.githubusercontent.com/716405/183017406-d3b7623a-c74f-4d04-b4bd-2586c7f52c85.png">

Link to report: https://validator.w3.org/nu/?doc=https%3A%2F%2Fwebplatform.news

**Comment by sideshowbarker** [comment]
OK, pinging @ylafon, since I can reproduce this at https://jigsaw.w3.org/css-validator/validator?uri=https://webplatform.news/ — so this issue isn’t specific to the HTML checker but instead is an upstream issue with the CSS-checking backend.

**Comment by ylafon** [comment]
Indeed, the definition of ratio in the css validator is way outdated. I opened an issue to track that.

**Comment by onlinebizsoft** [comment]
@ylafon do we have any roadmap for this?

**Comment by jaimeiniesta** [comment]
Hi, this is still failing, values like `aspect-ratio: 1`, `aspect-ratio: 1.5` or `aspect-ratio: 2.666666667` will fail but all will pass if we explicitly add the height denominator: `aspect-ratio: 1/1`, `aspect-ratio: 1.5/1`, etc., which shouldn't be needed as the spec says it defaults to 1.

<img width="790" alt="Captura de pantalla 2023-12-07 a las 10 29 42" src="https://github.com/validator/validator/assets/2629/597b32de-573b-478e-a298-f8bacfecbf67">

**Comment by oldrup** [comment]
Let me just add, that  `aspect-ratio: 1;` is also how core WordPress outputs its square 1:1 aspect ratio, meaning the editor or content creator has no influence on the markup, and can not, for instance, just enter 1:1 (instead of just 1), to satisfy the validator.

Meaning all square aspect ratios, often used for thumbnails, headshots etc in WordPress, will cause false positive when validating.

That, again, means we've got not choice but to tell people to ignore the errors in the validator, and that is a slippery slope, as valid HTML is a cornerstone of accessible web development.

🤔

**Comment by ylafon** [comment]
Just to note that it was fixed upstream (see right above).

**Comment by j9t** [comment]
@simevidas, confirmed fixed?

**Comment by simevidas** [comment]
@j9t Yes, seems so. I’m closing the issue.

https://validator.w3.org/nu/?showsource=yes&useragent=Validator.nu%2FLV+https%3A%2F%2Fvalidator.w3.org%2Fservices&acceptlanguage=&doc=https%3A%2F%2Foutput.jsbin.com%2Fnibevav%2Fquiet
